### PR TITLE
Favourite Event - animated button

### PR DESCRIPTION
**Pull request by RGBboy**
First part of [Trello 53 - Favourite Events](https://trello.com/c/EGwManv3/53-events-listing-flagging-favourite-events). This PR adds Lottie and a very simple animated button to the Event Details screen. Further functionality will be added in later PRs.

I have tested locally on iOS and Android. I did face an issue with iOS which now seems to be resolved. However I would like another dev to pull this and try run `yarn run-ios` and report back any problems.

It would also be good to check more devices so that we can confirm this lib is going to give us what we need.

## Tester check-list

* [ ] Tested on multiple devices / OS versions (Test on the physical device(s) you have access to, otherwise use BrowserStack):

  * [x] Galaxy S8 (Android 7.0/8.0)
  * [x] Galaxy S7 (Android 6.0)
  * [x] iPhone X (iOS 11.x)
  * [x] iPhone 8 (11.x)
  * [x] iPhone 7 (iOS 10.x)

Optional:

* [x] Google Pixel 2
* [x] Galaxy S8 Plus
* [ ] Galaxy S7 Edge
* [x] Galaxy S6
* [x] iPhone 7+
* [x] iPhone 6

### CODE REVIEW

**Comment by declanslevin** [comment]
Tested on physical Galaxy S8 and iPhone 7+/6, the rest were BrowserStack devices and the animation is working correctly on everything tested

File: android/app/build.gradle [hunk 1]
```diff
@@ -222,6 +222,7 @@ repositories {
 }
 
 dependencies {
+    compile project(':lottie-react-native')
```
Comment (frigus02) [inline android/app/build.gradle]: Can you change this `compile` to `implementation`? Unfortunately react native still generates the older gradle syntax (`compile`), so we need to manually change this.

File: src/screens/EventDetailsScreen/component.js [hunk 2]
```diff
@@ -20,7 +21,6 @@ import text from "../../constants/text";
 import type { Event, LocalizedFieldRef } from "../../data/event";
 import locale from "../../data/locale";
 import chevronLeftWhite from "../../../assets/images/chevron-left-white.png";
-import heartWhite from "../../../assets/images/heart-white.png";
```
Comment (frigus02) [inline src/screens/EventDetailsScreen/component.js]: Can we delete this icon or do we still use it somewhere else?

**Comment by frigus02** [comment]
This looks amazing. But it feels like it takes a while until the animation starts.

File: src/screens/EventDetailsScreen/component.js (hunk 2 above)
Comment (RGBboy) [reply src/screens/EventDetailsScreen/component.js]: It can go. Nice catch.

**Comment by RGBboy** [comment]
Yeah, I think there will need to be some tweaks. This PR was more proof that we could get this working across platforms. I will add a task to tweak the animations once this is all in.

File: android/app/build.gradle (hunk 1 above)
Comment (RGBboy) [reply android/app/build.gradle]: done

**Pull request merged**